src/platforms/python/integrations/logging/index.mdx

```diff
@@ -61,12 +61,17 @@ import logging
 import sentry_sdk
 from sentry_sdk.integrations.logging import LoggingIntegration
 
+# The SDK will honor the level of the logger, which is WARNING by default.
```

Comment:
Is this correct?

- The default level for python's `logging` library is `INFO`
- The default level for Sentry's `LoggingIntegration`'s `level` param is `INFO`
- The default level for Sentry's `LoggingIntegration`'s `event_level` param is `ERROR`

It's a little confusing what default level this comment refers to.

Reply:
Maybe rephrase to 

```suggestion
# The SDK will honor the level set by the logging library, which is WARNING by default.
```

assuming my understanding is correct.